docs/concepts/composition.md

```diff
@@ -495,6 +527,12 @@ spec:
     - type: ToCompositeFieldPath
       fromFieldPath: "status.atProvider.fullyQualifiedDomainName"
       toFieldPath: "status.address"
+
+    # It is also possible to combine multiple values from the composed resource
```

Comment:
Can you add an example for that, too?

Reply:
Added. Was struggling to find a good example, but I think exposing a DSN based on the `administratorLogin` input and the `fullyQualifiedDomainName` status is a topical usage of the combine patch 👍 Thanks for the URL suggestion, I think this is along the same lines.

Reply:
Yeah that's a great example!